src/fides/api/alembic/migrations/versions/a1e23b70f2b2_add_notice_and_exp_translations.py

```diff
@@ -201,7 +202,9 @@ def upgrade():
     )
     op.add_column(
         "privacyexperienceconfighistory",
-        sa.Column("language", sa.String(), nullable=False),
+        sa.Column(
+            "language", sa.String(), server_default="en", nullable=False
```

Comment:
want to review the approach here - could either be a server default or we could relax the nullable constraint, but then i'd assume we still need to make an assumption (in the data migration?) that the value should be english? there may very well be good reasons to avoid assuming a server default, so i'd be open to that!

Reply:
Great point yes, we need to go through every column where I'm saying "nullable=False" and do one of your two options above. I missed a few on the first pass.

 For this, I'd lean towards relaxing the nullable constraint, since there's also a unique constraint on this table of language 'x' experience_config_id.